src/gui/mzroll/mainwindow.cpp

```diff
@@ -2509,6 +2518,14 @@ void MainWindow::createToolBars() {
 	ionizationModeLabel->setFrameShape(QFrame::Panel);
 	ionizationModeLabel->setFrameShadow(QFrame::Raised);
 	
+	check_polly_login = new QLabel(hBox);
+	check_polly_login->setToolTip("Status of connection to Polly");
```

Comment:
use short tool tip message something like - Polly Connection.

Reply:
Changing it to Polly connection status..